internal/server/node/golden/get_property_labels/property-labels-class.json

```diff
@@ -1,18 +1,20 @@
 {
-  "Class": {
-    "inLabels": [
-      "domainIncludes",
-      "rangeIncludes",
-      "subClassOf",
-      "typeOf"
-    ],
-    "outLabels": [
-      "isPartOf",
-      "name",
-      "provenance",
-      "sameAs",
-      "subClassOf",
-      "typeOf"
-    ]
+  "data": {
```

Comment:
This is non-ID golden test.  Does it mean the response is changing?

Reply:
This is the case that the origin golden data is half json / half proto. And this is not the actual response sent in the API (actual one is encoded json even currently). So is only to serve as a diff checking.